flink-runtime/src/test/java/org/apache/flink/runtime/executiongraph/utils/SimpleSlotProvider.java

```diff
@@ -113,7 +114,11 @@ public SimpleSlotProvider(JobID jobId, int numSlots, TaskManagerGateway taskMana
 				return CompletableFuture.completedFuture(result);
 			}
 			else {
-				return FutureUtils.completedExceptionally(new NoResourceAvailableException());
+				if (allowQueued) {
```

Comment:
I prefer

```
} else if (allowQueued) {
...
} else {
...
}
```

because it decreases the level of nesting.

Reply:
Done.